docs/ui-ingestion.md

```diff
@@ -0,0 +1,238 @@
+
+![](./imgs/running-ingestion.png)
+
+If ingestion has executed successfully, you should see it's state shown in green as `Succeeded`. 
+
+![](./imgs/successful-ingestion.png)
+
+
+### Cancelling an Ingestion Run
+
+If your ingestion run is hanging, there may a bug in the ingestion source, or another persistent issue like exponential timeouts. If these situations, 
+you can cancel ingestion by clicking **Cancel** on the problematic run.
+
+![](./imgs/cancelled-ingestion.png)
+
+Once cancelled, you can view the output of the ingestion run by clicking **Details**. 
+
+
+### Debugging a Failed Ingestion Run
+
+![](./imgs/failed-ingestion.png)
+
+A variety of things can cause an ingestion run to fail. Common reasons for failure include:  
+
+1. **Recipe Misconfiguration**: A recipe has not provided the required or expected configurations for the ingestion source. You can refer
+   to the [Metadata Ingestion Framework](https://datahubproject.io/docs/metadata-ingestion) source docs to learn more about the configurations required for your source type.
+   
+2. **Failure to resolve Secrets**: If DataHub is unable to find secrets that were referenced by your Recipe configuration, the ingestion run will fail. 
+   Verify that the names of the secrets referenced in your recipe match those which have been created. 
+   
+3. **Connectivity / Network Reachability**: If DataHub is unable to reach a data source, for example due to DNS resolution
+   failures, metadata ingestion will fail. Ensure that the network where DataHub is deployed has access to the data source which
+   you are trying to reach. 
+   
+4. **Authentication**: If you've enabled [Metadata Service Authentication](https://datahubproject.io/docs/introducing-metadata-service-authentication/), you'll need to provide a Personal Access Token 
+    in your Recipe Configuration. To so this, set the 'token' field of the sink configuration to contain a Personal Access Token:
+   ![](./imgs/ingestion-with-token.png)
+
+The output of each run is captured and available to view in the UI for easier debugging. To view output logs, click **DETAILS** 
+on the corresponding ingestion run. 
+
+![](./imgs/ingestion-logs.png)
+
+## FAQ
+
+### I tried to ingest metadata after running 'datahub docker quickstart', but ingestion is failing with 'Failed to Connect' errors. What do I do?
+
+If not due to one of the reasons outlined above, this may be because the executor running ingestion is unable
+to reach DataHub's backend using the default configurations. Try changing your ingestion recipe to make the `sink.config.server` variable point to the Docker
+DNS name for the `datahub-gms` pod: 
+
+![](./imgs/quickstart-ingestion-config.png)
+
+### I see 'N/A' when I try to run ingestion. What do I do?
+
+If you see 'N/A', and the ingestion run state never changes to 'Running', this may mean 
+that your executor (`acryl-datahub-actions`) container is down. 
```

Comment:
should this be `datahub-actions` container? since that is the name of the container